flink-end-to-end-tests/test-scripts/test_streaming_file_sink.sh

```diff
@@ -29,6 +29,8 @@ echo "Executing test with ${OPENSSL_LINKAGE} openSSL linkage (random selection b
 s3_setup hadoop
 set_conf_ssl "mutual" "OPENSSL" "${OPENSSL_LINKAGE}"
 set_config_key "metrics.fetcher.update-interval" "2000"
+# this test relies on global failovers
+set_config_key "jobmanager.execution.failover-strategy", "full"
```

Comment:
The comma between key and value is not needed, it would result in a wrong config key.